src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -110,72 +152,136 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         }
     });
 
+    // Base slider grade
     fluid.defaults("fluid.slider", {
         gradeNames: ["fluid.viewComponent"],
         range: {}, // should be used to specify the min, max range e.g. {min: 0, max: 100}
-        selectors: {
-            thumb: ".ui-slider-handle"
-        },
         members: {
             combinedSliderOptions: {
                 expander: {
                     funcName: "fluid.slider.combineSliderOptions",
                     args: ["{that}.options.sliderOptions", "{that}.options.range"]
                 }
```

Comment:
Yeah, I think this makes sense - looking at it more closely `combinedSliderOptions` is necessary because the jQuery UI slider requires a single options block when being created. There's no need for it in the native implementation. Will refactor.